src/Microsoft.ML.AutoML/AutoMLExperiment/TunerFactory.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.ML.SearchSpace.Tuner;
+
+namespace Microsoft.ML.AutoML
+{
+    internal interface ITunerFactory
+    {
+        ITuner CreateTuner(TrialSettings settings);
+    }
+
+    internal class CfoTunerFactory : ITunerFactory
```

Comment:
>CfoTunerFactory [](http://example.com/codeflow?start=19&length=15)

I think it would be better to have the full name here so its more descriptive about what the class actually is.

Reply:
Sure

Reply:
Did you add the full name here? is it just Cfo? If so, could you add a comment where we can learn more about it cause that isn't much info to know what it does.